scripts/ci/qemu-run.sh

```diff
@@ -37,11 +37,13 @@ This is the QEMU wrapper script for the Dasharo Open Source Firmware Validation.
 
   Environmental variables:
     DIR         working directory, defaults to current working directory
+    DRIVE_PATH  path to file which will be added as a virtual hard drive
```

Comment:
@PLangowski we could add here info which kind of file need to be provided here, probably iso? or wic.gz from Yocto builds

Reply:
Added here dd64f2bfd25890be7dc7417d3df91a412e0fdddc reference to QEMU documentation because the list of available image file formats is too big.